KioskApplication/src/KioskApplication/database/util/CSVFileUtil.java

```diff
@@ -186,6 +186,16 @@ public static void writeNodesCSV(String path) {
                 NodeType nt = NodeType.values()[rs.getInt("nodeType")];
                 String nodeType = "";
 
+                if(global) {
+                    if(!(rs.getString("NodeID").startsWith("W"))) {
+                        continue;
+                    }
+                } else {
+                    if(rs.getString("NodeID").startsWith("W")) {
+                        continue;
+                    }
+                }
+
```

Comment:
This is like, a serious hack. I'll approve this for the sake of finishing iteration 1 but we seriously need to do something better than this in the future.